flow/designs/asap7/aes/config.mk

```diff
@@ -1,52 +1,25 @@
-$(info [INFO-FLOW] AES Design)
+export PLATFORM               = asap7
 
-DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
-DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+#User can modify below variables as per design requirement
+export CORNER                ?= BC
 
 export DESIGN_NAME            = aes_cipher_top
 export DESIGN_NICKNAME        = aes
-export DESIGN                 = aes
-
-export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
-export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
-export ABC_AREA               = 1
-
-export CORNER                ?= BC
-
-export LIB_FILES             += $($(CORNER)_LIB_FILES)
-export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
-export DB_FILES              += $($(CORNER)_DB_FILES)
-export DB_DIRS               += $($(CORNER)_DB_DIRS)
-export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
-export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
-export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
-
-export ABC_CLOCK_PERIOD_IN_PS = 400
-
-export DESIGN_POWER           = VDD
-export DESIGN_GROUND          = VSS
 
-export DESIGN_DIR
+export VERILOG_FILES = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
+export ABC_AREA               = 1
 
-export PLATFORM               = asap7
-export DESIGN_PDK_HOME
-
-export CORNER                ?= BC
-
-export PDN_CFG                = $(FOUNDRY_DIR)/openRoad/pdn/grid_strategy-M2-M5-M7.cfg
-
+#Floorplaning and placement variables for relative die size
 export CORE_UTILIZATION       = 30
 export CORE_ASPECT_RATIO      = 1
 export CORE_MARGIN            = 2
 export PLACE_DENSITY          = 0.60
 
-export DONT_USE_SC_LIB         = $(OBJECTS_DIR)/lib/merged.lib
-
 ifeq ($(USE_4X),1)
```

Comment:
what is this for?

Reply:
are you pointing USE_4X?
If yes, in ASAP7 there is 1X and 4X cells. We need to check with ASAP7 about in details for this usage.

Reply:
@vijayank88 This question is for DONT_USE_SC_LIB

Reply:
Why is this relevant only for a single design?  This won't do anything as you have already defined CORE_UTLIZATION which has higher precedence.

Reply:
@dralabeing DONT_USE_SC_LIB moved to platform config.mk. highlight in red is removed.

@maliberty aes,ibex,jpeg,gcd are built-in design. I have cleaned up duplicate variables only, if I modify configuration parameter CI will not pass as its affects metadata checks.

I thinks ASAP7 contributor intention is user has to know 7nm library has 1x, 4x cells. Example format shown part of design contribution. So new user can utilize this as well.

If you want to remove this confirm me I can remove it.

Reply:
I don't think the 4x is useful. This was only part of the PDK to work around commercial tool license restrictions. OpenROAD has no such restrictions.

Reply:
@rovinski then I will clean up all 4X die area definition in config.mk

Reply:
removed 4X usage from config.mk